content/reference-rippled.md

```diff
@@ -951,7 +1130,7 @@ Each object in the `transactions` array, if present, may contain any or all of t
 ## account_lines ##
 [[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/AccountLines.cpp "Source")
 
-The `account_lines` method returns information about the account's lines of trust, including balances in all non-XRP currencies and assets. All information retrieved is relative to a particular version of the ledger.
+The `account_lines` method returns information about an account's trust lines, including balances in all non-XRP currencies and assets. All information retrieved is relative to a particular version of the ledger.
```

Comment:
Per the [style guide](https://ripplelabs.atlassian.net/wiki/display/DOCS/Ripple+Documentation+Style+Guide) we do not hyphenate trust lines.